src/actions/query.js

```diff
@@ -355,8 +356,15 @@ export function executeQuery(
 			props,
 			internalValues,
 		} = getState();
-		const lockTime = config.initialQueriesSyncTime;
-		const initialTimestamp = config.initialTimestamp;
+		let lockTime = config.initialQueriesSyncTime;
+		let initialTimestamp = window.dummyTime;
+
+		// override logic for locking queries for a period of time
+		if (config.queryLockConfig instanceof Object) {
```

Comment:
@mohdashraf010897 Only apply lock mechanism when values are set by `setSearchState` action. We don't want 100ms delay for normal queries.

Reply:
@bietkul Yes, It's only done for `setSearchState` action. After the query I am resetting the config so other queries aren't affected.

<img width="418" alt="image" src="https://user-images.githubusercontent.com/57627350/182907851-ebcff6c7-4c70-4722-b7a7-7c488fc4e8c5.png">